src/Framework/Framework/Binding/DotvvmProperty.cs

```diff
@@ -345,6 +345,8 @@ public static DotvvmProperty[] ResolveProperties(Type type)
             return registeredProperties.Values.Where(p => types.Contains(p.DeclaringType)).ToArray();
         }
 
+        public static IEnumerable<DotvvmProperty> AllProperties => registeredProperties.Values;
+
```

Comment:
This is a duplicate of `GetRegisteredProperties` introduced by property aliasing. I don't care which version survives (`AllProperties` is shorter, so I'd actually prefer it), just please don't forget to add `AllAliases` when you merge `main` into this branch. Thanks. :)

Reply:
Done, would please have another look?